src/bptl/work_units/zgw/tasks.py

```diff
@@ -264,6 +264,7 @@ class RelatePand(ZGWWorkUnit):
     **Required process variables**
 
     * ``zaakUrl``: URL reference to a ZAAK in a Zaken API. The PANDen are related to this.
+    * ``panden``: list of URL references to PANDen in BAG API.
```

Comment:
Left over from elsewhere?

Reply:
It is a task from Utrecht hackathon. I just spotted that it lacks a description for one of variables